mix.exs

```diff
@@ -18,6 +18,7 @@ defmodule Honeybadger.Mixfile do
 
   def application do
     [applications: [:httpoison, :logger],
+     env: [environment_name: Mix.env],
```

Comment:
This looks safe to me. I'm at the point where the only place I'm willing to guarantee Mix exists is in the Mixfile itself :-p